proposed/version-support.md

```diff
@@ -0,0 +1,111 @@
+also contains new features or functionality but that is not required.
+
+* **Minor version** - A minor version of a project is a release that is documented
+as containing new features or functionality but not API breaks.  Typically a minor 
+version also contains bug fixes but that is not required.
+
+* **Patch version** - A patch version of a project is a release that is documented
+as containing bug fixes but no new features or API changes.
+
+* **Support** - A given release of a project *supports* a given PHP release if
+it executes without parse errors and performs its designated task as expected.
+It is not required to leverage any particular language feature or syntax, only
+that the code executes without error.
+
+* **Not Support** - A given release of a project that *does not support* a given
+PHP release if no effort is make to ensure that it executes without parse errors
+or performs as expected.  While the code may execute correctly, that is considered
+to be incidental.  A project SHOULD NOT attempt to break compatibility with a given
+PHP version simply for its own sake; it should only not consider that PHP version
+either way when determining what language features to use.
+
+* **Offer** - A public web host *offers* a PHP version if its customers are able
+to host their application using that PHP version on all hosting plans that offer
+PHP, regardless of cost.  A web host MAY require a user to select a non-default
+option in a control panel or similar, but MUST NOT require a custom configuration
+or other special arrangements with the host.  This requirement applies to both
+new and existing accounts.
+
+* **Participating Project** - A *participating project* is a PHP project that
+openly and publicly declares its intent to comply with the expectations of this 
+standard.  A participating project MUST make reasonable effort to comply with
+the expectations of this standard.
+
+* **Participating Host** - A *participating host* is a company that offers web
+hosting to customers that includes the ability to execute PHP code and openly and
+publicly declares its intent to comply with the expectations of this standard. A
+participating host MUST make reasonable effort to comply with the expecatations of
+this standard.
+
+Such hosts MAY be specific to a particular project or projects, or MAY offer
+general hosting for arbitrary PHP code. Such hosts MAY offer "shared hosting",
+virtual machines, dedicated hardware, cloud configurations, or any other form of
+hosting.  This standard applies to all offerings that include the ability to
+execute PHP code, and applies to all offered PHP SAPI configurations.  (For instance,
+it applies to both command line PHP and web scripts.)
+
+
+## Expectations of projects
+
+A Major Release of a participating project MUST:
+
+* support the most recent stable Minor PHP release older than thirty (30) days.
+
+* support the most recent stable Major release older than six (6) months.
```

Comment:
On the flipside, though, one of the Drupal release managers actually asked to include the inverse. Mainly, because there is no PHPUnit release now that is supported on both 5.5 and 7, which is a major epic fail for any project that wants to work on 5.5, 5.6, and 7 (like, say, Drupal).  I don't know that this document is the right way to solve that, but it's a legit concern.